prebid-mobile/pbm-api/android/pbm-plugin-renderer.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: page_v2
+title: Plugin Renderer - Android
+description: Guide to implement a plugin renderer
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile
+sidebarType: 2
+---
+
+# Prebid Plugin Renderer
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+Plugin Renderer is a feature that enables the ability to delegate the ad rendering to a component of yours. Such feature turn possible, for instance, the rendering of non-standard ad responses that Prebid Mobile SDK can not render by itself. This integration require from you, in first place, to have a Bidder Adapter implemented ([see more](prebid-server/developers/add-new-bidder-go.html)) in order to handle bid requests from the Prebid Mobile SDK that include your plugin renderer.
+
+![Plugin Renderer big picture](/assets/images/prebid-mobile/prebid-plugin-renderer.png)
+
+### Ad view transposing
+
+Everytime that a bid response is received and it reaches the rendering stage, Prebid SDK will delegate the ad view rendering to an existing plugin renderer, such as a custom one if this is elected or the default one in any other case.
+
+Take the example on the image below where a BannerView will have its ad view transposed accordingly to the plugin renderer status. That way an inner view which represents the ad itself is handled totally under the hood from the publisher point of view, what makes unnecessary any change on the BannerView loading or initialization.
+
+> In case of Interstitial ad this is just inflated in the foreground regardless the view hierarchy.
```

Comment:
This does not what you expect, even it's markdown standard.

I would recommend using

```suggestion
{: .alert.alert-info :}
In case of Interstitial ad this is just inflated in the foreground regardless the view hierarchy.
```